apps/e2e-tests/tests/example.spec.ts

```diff
@@ -0,0 +1,53 @@
+import { expect, test } from '@playwright/test';
+import { appName, urls } from '../configuration';
+import { checkIfAppIsAvailable } from '../utils/check-if-app-is-available';
+import { installTheApp } from '../utils/install-the-app';
+import { logInIntoDashboard } from '../utils/log-in-into-dashboard';
+import { randomString } from '../utils/random-string';
+
+test("The app can be installed", async ({page}) => {
+  // TODO: add function to completely remove all the installed apps in the test env
+  // alternative: make locators tighter to restrict it only to the tested app
+  await logInIntoDashboard({page})
+  await installTheApp({page})
+  await checkIfAppIsAvailable({page})
+})
+
+test("Smoke test of the configuration", async ({page}) => {
+  await logInIntoDashboard({page})
+
+  // open app configuration view
+  await page.goto(urls.saleor.dashboard.apps, {timeout: 20000, waitUntil: "load"}); 
+  await page.getByText(appName).first().click()
+
+  // generate unique marker to ensure values are updated during this test run
+  const marker = randomString()
+
+  // fill the configuration fields
+  await page.frameLocator('iframe').locator('input[name="secretKey"]').click();
+  await page.frameLocator('iframe').locator('input[name="secretKey"]').fill('secret-key-'+marker);
+  await page.frameLocator('iframe').locator('input[name="searchKey"]').click();
+  await page.frameLocator('iframe').locator('input[name="searchKey"]').fill('search-key-'+marker);
+  await page.frameLocator('iframe').locator('input[name="appId"]').click();
+  await page.frameLocator('iframe').locator('input[name="appId"]').fill('app-id-'+marker);
+  await page.frameLocator('iframe').locator('input[name="indexNamePrefix"]').click();
+  await page.frameLocator('iframe').locator('input[name="indexNamePrefix"]').fill('prefix-'+marker);
```

Comment:
tests are specific to search app. so there is only installing and opening the app which is shared behavior for every app. 

so we need to figure out how we will approach tests specific to the app

We can try to run some global scripts, but why don't we add e2e script to every app folder? playwright config etc can be in shared package

then its easy, because turobrepo will detect changes and run tests accordingly to cache, it will be easy to run for every package and to filter 

we can share common assertions in e2e package too